groovy/gradle/src/org/netbeans/modules/gradle/spi/nodes/NodeUtils.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.netbeans.modules.gradle.spi.nodes;
+
+import java.awt.Image;
+import javax.swing.Icon;
+import javax.swing.UIManager;
+import org.netbeans.api.annotations.common.StaticResource;
+import org.openide.util.ImageUtilities;
+
+/**
+ *
+ * @author mkleint
```

Comment:
Copy based programming again.

Reply:
Plugin Mentality. Maven project use the same code. Could be added somewhere else later.